src/wp-admin/js/customize-controls.js

```diff
@@ -4151,57 +4155,84 @@
 			var control = this,
 				isHueSlider = this.params.mode === 'hue',
 				updating = false,
-				picker;
+				picker,
+				__ = wp.i18n.__;
```

Comment:
ClassicPress currently does not include the [i18n package](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-i18n/), so we need to consider if we begin to bundle it, or revert to using `wp_localize_script()`.

Reply:
`__ = wp.i18n.__` is already used in `wp-admin/js/color-picker.js`. In fact, I just did a quick `grep` of the `wp-admin/js` folder, and found it's used in all the following files:
```
user-profile.js
site-health.js
password-strength-meter.js
postbox.js
post.js
privacy-tools.js
edit-comments.js
common.js
theme-plugin-editor.js
updates.js
image-edit.js
color-picker.js:
```
In `wp-includes/js` there is also the `media-views.js` file.

So I think we need to include that package or else we're going to have a lot of files to re-write.